src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -116,6 +116,7 @@ const InputPassword: UISchemaElementComponent<UISchemaElementComponentWithValida
           'aria-describedby': ariaDescribedByIds,
           ...attributes,
         }}
+        className={noTranslate ? 'no-translate' : undefined}
```

Comment:
Is this needed for the automation suite? Does it fail when it notices characters in the field that are no OK-PL translated?

Reply:
the show functionality will cause english leak. So added it before someone creates a ticket. :)

Reply:
Ahh, makes sense! Ty!